### DISCUSSION TITLE
Google Search Console returned error even if I added `trailingSlash: false`

### DISCUSSION THREAD

**Discussion by tamagoez** [facebook/docusaurus#6706] [Q&A] [answered]
I have few knowledge, please forgive me.

I tried to clore my Docusaurus page with `sitemap.xml` generated by preset using Google Search Console.
Here's a part of code in `docusaurus.config.js` and `vercel.json`.
```
# docusaurus.config.js

  trailingSlash: false,

^^^^^^^^^^^Omission^^^^^^^^^^^

  presets: [
    [
      '@docusaurus/preset-classic',
      {
        sitemap: {
          changefreq: 'hourly',
          priority: 0.5,
        },
      },
    ],
```

```
# vercel.json
{
  "trailingSlash": false
}
```

But Google Search Console returned error even if I added `trailingSlash: false`.
```
Sitemap can be read, but has errors
URL not allowed

This url is not allowed for a Sitemap at this location.
Examples
URL:
https://~~~~.vercel.app/blog
Line 1
```

This is `sitemap.xml` what generated.
```xml
<urlset xmlns="http://www.sitemaps.org/schemas/sitemap/0.9" xmlns:news="http://www.google.com/schemas/sitemap-news/0.9" xmlns:xhtml="http://www.w3.org/1999/xhtml" xmlns:image="http://www.google.com/schemas/sitemap-image/1.1" xmlns:video="http://www.google.com/schemas/sitemap-video/1.1">
<url>
<loc>https://~~~~.vercel.app/blog</loc>
<changefreq>hourly</changefreq>
<priority>0.5</priority>
</url>
</urlset>
```

**Comment by slorber** [comment, marked as answer]
@tamagoez it's hard for us to tell you what is wrong if you hide the real site URL and we can't inspect your deployment

At least tell us what's the response from your host for https://~~~~.vercel.app/blog

---

We don't have trailing slashes here and it seems to work fine 🤷‍♂️ 

https://docusaurus.io/sitemap.xml

<img width="1386" alt="image" src="https://user-images.githubusercontent.com/749374/154492668-4d2c703c-57a4-4120-b7bf-da70564dfd6d.png">


---

In my experience, Vercel has some non-std way to serve static files compared to other hosts.

https://github.com/slorber/trailing-slash-guide

<img width="596" alt="image" src="https://user-images.githubusercontent.com/749374/154493204-3ce1fe27-7785-478e-bd14-f0f46e8de173.png">

I suggest trying to use `cleanUrls: true` too

**Reply by slorber** [reply]
We don't document `cleanUrls: true` for Vercel in our docs but we probably should: https://docusaurus.io/docs/deployment#deploying-to-vercel

See also https://github.com/slorber/trailing-slash-guide/issues/1

**Reply by tamagoez** [reply]
https://vividarmy-wikinotes.vercel.app/
and also
https://github.com/tamagoez/vividarmy-wikinotes

https://vividarmy-wikinotes.vercel.app/sitemap.xml
is my sitemap URL.

**Reply by slorber** [reply]
@tamagoez you probably have configured a wrong `siteConfig.url` for your site

- Real blog URL: https://vividarmy-wikinotes.vercel.app/blog
- Sitemap blog url: https://wikinotes-vividarmy.vercel.app/blog

**Reply by tamagoez** [reply]
Oh, yes.
It was my overlapse...
Sorry for my careless miss.
After I check Google Search Console workings fine, I'm going to close this discussions.
Thank you.
P.S. I could check sitemap loaded succesfully. Thank you so much.
